infra/stacks/unfurl-service/unfurl-service.ts

```diff
@@ -222,7 +221,7 @@ export class UnfurlService extends pulumi.ComponentResource {
       { parent: this }
     );
 
-    this.domain = `https://${SERVICE_DOMAIN_NAME}`;
+    this.domain = getServiceUrl(ServiceUrl.UNFURL_SERVICE_URL);
```

Comment:
This is out of scope but we are incorrectly calling a URL a domain in all of these